### DISCUSSION TITLE
launch the project using macOS ( i have crossover )

### DISCUSSION THREAD

**Discussion by Hakimbhb** [YimMenu/YimMenu#1310] [Q&A]
hey friend i m using crossover to launch gpa v but it doesn't  support visual studio code  

crossover could launch many .exe file on macOS and i don't wanna see parallels bus i only have 256 gb

my question is:  
how could i debug the source code i wanna try new feature to it 
i m new and noob in that i never launch a project in C# and as i m macOS user idk if it possible i try using make on macOS but it have problem with windows libaries it seems like it s from windows directly anyone have tried or know how could i do that ?? 

thank u  lot for answering

**Comment by Yimura** [comment]
It's best to use a Windows VM or in your case crossover, I'm not aware of any other methods.

Also the project is in C++ not C#.

**Reply by Hakimbhb** [reply]
thank u for making thing more clear such as i mention i m searching for methods without VM  i wish that someone have a solution!!

and for crossover it didn't t support visual studio code yet so it there is another app i could use it will help!
